2023-review.md

```diff
@@ -0,0 +1,165 @@
+  `97 hours` spent / `3.7 hour` tutorial = `26.2` wh/vh
+* [API Platform Ep2](https://symfonycasts.com/screencast/api-platform-security)
+  `83 hours` spent / `3.7 hour` tutorial = `22.4` wh/vh
+* [API Platform Ep1](https://symfonycasts.com/screencast/api-platform)
+  `58 hours` spent / `2.9 hour` tutorial = `20.4` wh/vh
+* [AssetMapper](https://symfonycasts.com/screencast/asset-mapper)
+  `37 hours` spent / `2.1 hour` tutorial = `17.6` wh/vh
+* [Integration Testing](https://symfonycasts.com/screencast/phpunit-integration)
+  `26.5 hours` spent / `1.0 hour` tutorial = `26.5` wh/vh
+* [Doctrine Queries](https://symfonycasts.com/screencast/doctrine-queries)
+  `25 hours` spent / `1.4 hour` tutorial = `17.8` wh/vh
+
+The key is ``wh/vh``: how much work goes into each hour of video produced.
+I have a few ideas to improve this:
+
+* **New tutorial authoring tool**. We use a fantastic, internal tool called
+  TutsHero to help us build the code for each tutorial. But it's time to
+  modernize that and better integrate it with the recording process. This
+  should also alleviate the time it takes to add our magic "code blocks".
+
+* **AI in Script Editing**. Fun fact: I first record rough audio while recording
+  the video. We use AI to transcribe that. Then we manually clean up the
+  script, reformatting code (`foo arrow bar` becomes `$foo->bar()`), adding
+  `ticks` around code, and rephrasing. Finally, I record this. The script
+  editing this takes a significant amount of time and I think AI can help
+  automate this.
+
+What about other authors? Sure! Though finding consistent, high-quality authors is
+hard. If you have a passion for teaching, definitely [reach out](https://symfonycasts.com/contact)!
+
+### 2023 SymfonyCasts.com
+
+Our team - Diego, Vladimir, Victor & Leanna - work a lot on SymfonyCasts.com so
+that we can have nice things... and I can focus on tutorials (Diego & Victor also
+help with creating tutorials).
+
+Out of the million different things from 2023, one stands our: our new Tailwind-powered
+design. We're rolling this out slowly - but you can see it on this blog section
+or any course pages. And this is more than a re-skinning: it's a chance for us
+to embrace the [LAST Stack](https://symfonycasts.com/blog/last-stack) principles:
+leverage Turbo, delete a lot of custom JavaScript and use Stimulus everywhere.
+It's *fantastically* fun to work with.
+
+## Comments & Questions Answered
+
+At the end of each tutorial, I like to say:
+
+> If you have any questions, we're here for you down in the comments.
+
+And we mean it. In 2023, 2572 comments were posted to the site, out of which
+1144 were replies from **us**. The best part? All of those questions and
+answers live in public: they help everyone.
+
+### What about 2024?
+
+So what's next? I have a few goals for 2024:
+
+* **More languages**. Each new tutorial's script and subtitles are already
+  translated into Spanish in a high-quality way thanks to AI. We should add
```

Comment:
Yes, but not... because we don't have a mechanism yet for users to do this. We need to create that mechanism first.

Reply:
Ah, ok, Then that's what we need to do: we should explain that both files need to be fixed (that's a bit more than just linking to a single file). And is that truly everything that's needed? When we merge that PR, will everything update?

Reply:
> And is that truly everything that's needed? When we merge that PR, will everything update?

Yes, we have a GitHub webhook, whenever you update a subtitles file it will update the file in S3, and on a course build we update course and chapter translations reading the original files from Github, so yes, everything will update :)

Reply:
We need a help tooltip aside from the GitHub link button explaining what files need to be updated